src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1828,6 +1827,16 @@ def _private_key_bytes(self, encoding, format, encryption_algorithm,
                 "Unsupported encoding for TraditionalOpenSSL"
             )
 
+        # OpenSSH + PEM
+        if format is serialization.PrivateFormat.OpenSSH:
+            if encoding is serialization.Encoding.PEM:
```

Comment:
Hmm. This is definitely PEM, but we also have that OpenSSH encoding that we use for public format. The inconsistency is a bit troubling, but I don't think I have a better suggestion

Reply:
I think in case of private keys, Encoding.OpenSSH should be reserved for binary (non-base64) version of same format.